Packs/FeedGCPWhitelist/Integrations/FeedGoogleIPRanges/FeedGoogleIPRanges.yml

```diff
@@ -0,0 +1,120 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Google IP Ranges Feed
+  version: -1
+configuration:
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- defaultvalue: All GCP customer global and regional external IP ranges
+  display: IP Address Ranges
+  hidden: false
+  name: ip_ranges
+  options:
+    - All GCP customer global and regional external IP ranges
+    - All available Google IP ranges
+  required: true
+  type: 15
```

Comment:
Why not make it multi-select?

Reply:
because `All GCP customer global and regional external IP ranges` is a subset of `All available Google IP ranges` so there is no point of picking 2 of them together.